src/lv_objx/lv_list.h

```diff
@@ -78,6 +78,13 @@ enum {
 };
 typedef uint8_t lv_list_style_t;
 
+/** List layouts. **/
```

Comment:
I'm being really picky with this, but the end comment can be `*/` not `**/`.

Reply:
Will fix it.